e2e/src/usecases/ChooseYourAdventure.js

```diff
@@ -8,8 +8,7 @@ export default ChooseYourAdventure = () => {
     await launchApp({
       newInstance: true,
       launchArgs: {
-        onboardingOverrides:
-          'EnableBiometry,ProtectWallet,PhoneVerification,CloudBackupSetup,CloudBackupRestore',
+        onboardingOverrides: 'EnableBiometry,ProtectWallet,PhoneVerification,CloudBackup',
```

Comment:
should we make a ticket to update tests to include `CloudBackupSetupInOnboarding`, since that's the default?

Reply:
https://linear.app/valora/issue/ACT-1471/update-e2e-tests-to-account-for-cab-in-onboarding